recipe/meta.yaml

```diff
@@ -27,7 +27,7 @@ requirements:
     - bzip2
     - krb5
     - libuuid
-    - libxml2 2.10
+    - libxml2 {{ libxml2 }}
     - mysql-connector-c
     - openssl 1.1.1
```

Comment:
@skupr-anaconda this seems to fail on linux

Reply:
We should rebuild `mysql-connector-c` against **openssl 3**:
`- package mysql-connector-c-6.1.11-hf4847fb_0 requires openssl >=1.0.2o,<1.0.3a, but none of the providers can be installed`

Reply:
I tried to rebuild mysql-connector-c https://github.com/AnacondaRecipes/mysql-connector-c-feedstock/pull/2 but it's blocked on Prefect because of `Error: HTTP 403 FORBIDDEN for url <https://dev.mysql.com/get/Downloads/Connector-C/mysql-connector-c-6.1.11-src.tar.gz>`

Reply:
mysql-connector-c has been fixed. 

I've pushed my commit to use openssl 3

Reply:
Now it uses openssl 3.